### DISCUSSION TITLE
Java support?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [tursodatabase/libsql#1633] [Q&A] [answered]
I don't see java/jdbc among the supported platforms, why?

**Comment by haaawk** [comment]
You can try https://github.com/libsql/libsql-client-java

**Comment by hrstoyanov** [comment]
Thanks! Cn you please list this on the home page along with the other language drivers?

**Reply by serge-rider** [reply, marked as answer]
You can try this JDBC driver:  https://github.com/dbeaver/dbeaver-jdbc-libsql
It is fully functional and it is in Maven Central.
